app/controllers/general.php

```diff
@@ -955,7 +958,7 @@ function router(App $utopia, Database $dbForConsole, callable $getProjectDB, Swo
 
         $response->dynamic(
             new Document($output),
-            $utopia->isDevelopment() ? Response::MODEL_ERROR_DEV : Response::MODEL_ERROR
+            $utopia->isDevelopment() || $hasDevelopmentKey ? Response::MODEL_ERROR_DEV : Response::MODEL_ERROR
```

Comment:
@lohanidamodar we shouldn't do that. This will expose sensitive Appwrite information to both devs and potentially to end users as well.

Reply:
How do we do it then, we want to expose more information on error when there's a development key, we need to decide what we want to expose and filter accordingly.

Reply:
Removed, we will add this feature with more consideration later